onnxruntime/core/providers/openvino/ov_interface.h

```diff
@@ -5,36 +5,52 @@
 
 #include <inference_engine.hpp>
 #if defined (OPENVINO_2022_1)
-#include <openvino/core/core.hpp>
-#include <openvino/runtime/runtime.hpp>
+#define OV_API_20
```

Comment:
why do we need an additional macro defined (OV_API_20)?
We have OpenVINO_2022_1 macro defined right. why do we have to change this.

Reply:
When OPENVINO_2022_2 comes in i will have to make changes in all files to adapt to New API. In this case I will just have to make changes in one place

Reply:
Yes makes sense.